src/api/services/TransactionAPI.ts

```diff
@@ -0,0 +1,60 @@
+/**
+ * Italia PagoPA Proxy
+ * Cittadinanza Digitale PagoPA services
+ */
+
+import * as express from "express";
+import fetch from "node-fetch";
+import querystring = require("querystring");
+import { CONFIG } from "../../Configuration";
+import { ITransactionListResponse } from "../types/TransactionResponse";
+
+// Transaction Service for PagoPA communications
+export class TransactionAPI {
+  // Retrieve transactions
+  public static getTransactionListResponse(
+    res: express.Response,
+    errorCallback: (response: express.Response, errorMsg: string) => void,
+    successCallback: (
+      response: express.Response,
+      transactionList: ITransactionListResponse
+    ) => void,
+    apiRequestToken: string,
+    id?: number,
+    start?: number,
+    size?: number
+  ): void {
+    let url; // tslint:disable-line
+    if (id !== undefined) {
+      url =
+        CONFIG.PAGOPA.HOST +
+        ":" +
+        String(CONFIG.PAGOPA.PORT) +
+        CONFIG.PAGOPA.SERVICES.TRANSACTION.replace(":id", String(id)) +
```

Comment:
it looks like `CONFIG.PAGOPA.SERVICES.TRANSACTION` should be a function `number => string`

Reply:
It's a good idea.
Done. It will be available into next Push